Kalyani/HTML/Assignment4/KalyaniA4_3.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Assignment4_3</title>
+
+<style>
+    table,th,td{
+        border:1px solid black;
+        
+    }
+</style>
+</head>
+<body>
+<table>
+    <tr>
+        <th rowspan="3">Day</th>
+        <th colspan="3">Seminar</th>
+        </tr>
+        <tr>
+            <th colspan="2">Schedule</th>
+            <th rowspan="2">Topic</th>
+        </tr>
+        <tr>
+        <th>Begin</th>
+        <th>End</th>
+        </tr>
+
+    </tr>
+    <tr>
+        <td rowspan="2">Monday</td>
+        <td bgcolor="lightpink" rowspan="2"style="text-align: center;">8:00am</td>
+        <td  bgcolor="lightblue" rowspan="2"style="text-align: center;">5:00pm</td>
+        <td style="text-align: center;">Introduction to XML</td>
+    </tr>
+    <tr>
+        <td style="text-align: center;">Validity DTD and Relax NG</td>
+    </tr>
+    <tr>
+        <td rowspan="3">Tuesday</td>
+        <td bgcolor="lightpink" style="text-align: center;">8:00am</td>
+        <td bgcolor="lightblue"style="text-align: center;">11:00am</td>
+        <td rowspan="1.5" style="text-align: center;">XPath</td>
+    </tr>
+    <tr>
+        <td  bgcolor="lightpink" style="text-align: center;">11:00am</td>
+        <td bgcolor="lightgreen" style="text-align: center;">2:00am</td>
+        <td rowspan="1.5" style="text-align: center;">XSL Transformations</td>
+    </tr>
```

Comment:
Try to make it as given in Question.
maybe float values for rowspan doesn't work
try using higher number of rowspan eg. rowspan =6

Reply:
It's Work .The output for 1.5 is same as that of the output 3

Reply:
decimal values for rowspan do not work
check your snack I have send screenshot of your code output on my machine.It does not look like what asked in question.

Reply:
i changed

Reply:
and changes committed now